src/component/chrome/overview-switch-container.tsx

```diff
@@ -0,0 +1,32 @@
+// import * as MobX from 'mobx';
+import { observer } from 'mobx-react';
+import { Project } from '../../store/project';
+import * as React from 'react';
+import { Store } from '../../store/store';
+import { ViewSwitch } from '../../lsg/patterns/view-switch';
+
+@observer
+export class OverviewSwitchContainer extends React.Component {
+	protected getName(): string {
+		const store = Store.getInstance();
+		const project: Project | undefined = store.getCurrentProject();
+		if (store.pageOverviewIsOpened) {
+			return 'Pages';
+		} else {
+			return project ? project.getName() : 'Unnamed Project';
+		}
+	}
+
+	public render(): JSX.Element {
+		const store = Store.getInstance();
+		return (
+			<ViewSwitch
+				onLeftClick={() => store.togglePageOverview()}
+				onRightClick={() => null}
```

Comment:
if you don't need that function always, please consider to make it optional

Reply:
✅